google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -122,14 +132,19 @@ def visit(self, resource):
             resource.get_billing_info(self.get_client())
             resource.get_enabled_apis(self.get_client())
             resource.get_kubernetes_service_config(self.get_client())
-
             self.write(resource)
         except Exception as e:
             LOGGER.exception(e)
             progresser.on_error(e)
+            attrs['exception'] = e
```

Comment:
@red2k18 not sure if adding the `exception` attribute has so much value in the end

Reply:
Removed